docs/sdk/SDK_API_DEFINITION.md

```diff
@@ -0,0 +1,205 @@
+# Mesh Data Plane SDK Definition
+
+## mesh_create_client()
+```c
+int mesh_create_client(MeshClient **mc,
+                       MeshClientConfig *cfg)
+```
+Creates a new mesh client from the given configuration structure.
+
+#### Parameters
+* `[OUT]` `mc` – Address of a pointer to a mesh client structure.
+* `[IN]` `cfg` – Pointer to a mesh client configuration structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_delete_client()
+```c
+int mesh_delete_client(MeshClient **mc)
+```
+Deletes the mesh client and its resources.
+
+#### Parameters
+* `[IN/OUT]` `mc` – Address of a pointer to a mesh client structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_create_connection()
+```c
+int mesh_create_connection(MeshClient *mc,
+                           MeshConnection **conn)
+```
+Creates a new media connection for the given mesh client.
+
+#### Parameters
+* `[IN]` `mc` – Pointer to a parent mesh client.
+* `[OUT]` `conn` – Address of a pointer to the connection structure.
+* `[IN]` `cfg` – Pointer to a connection configuration structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_apply_connection_config_memif()
+```c
+int mesh_apply_connection_config_memif(MeshConnection *conn,
+                                       MeshConfig_Memif *cfg)
+```
+Applies the configuration to setup a single node direct connection via memif
+
+#### Parameters
+* `[IN]` `conn` – Pointer to a connection structure.
+* `[IN]` `cfg` – Pointer to a configuration structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_apply_connection_config_st2110()
+```c
+int mesh_apply_connection_config_st2110(MeshConnection *conn,
+                                        MeshConfig_ST2110 *cfg)
+```
+Applies the configuration to setup an SMPTE ST2110-xx connection via Media Proxy
```

Comment:
"." at the end of sentence.

Reply:
Fixed in eae8b083e0210567994eb379b5e228742451c44b.